models/Movie.model.js

```diff
@@ -0,0 +1,20 @@
+// IMPORTACIONES
+const mongoose = require("mongoose")
+
+
+// SCHEMA
+const movieSchema =  mongoose.Schema({ //requisitos para crear un libro
+    title: String,
+    director: String,
+    stars: [String],
+    image: String,
+    description: String,
+    showtime: [String]
```

Comment:
No te aparecen por que la propiedad en tu DB es showtime, y tu seeeds tiene showtimes 👀 
```suggestion
    showtimes: [String]
```